bpf/bpf_lxc.c

```diff
@@ -155,6 +155,15 @@ static __always_inline int ipv6_l3_from_lxc(struct __ctx_buff *ctx,
 	if (!revalidate_data(ctx, &data, &data_end, &ip6))
 		return DROP_INVALID;
 
+	/* When an endpoint connects to itself via service clusterIP, we need
```

Comment:
Re: forward port to master

If `*dstID` is not populated, then we may end up dereferencing a `null` pointer at in case of [host firewall related processing](https://github.com/cilium/cilium/blob/master/bpf/bpf_lxc.c#L343)? @pchaigno please check. I checked other references in the policy verdict notification, and those are already wrapped in `hairpin_flow` check.